src/v2/Apps/Artist/Components/ArtistTopWorksRail/__tests__/ArtistTopWorksRail.jest.tsx

```diff
@@ -29,15 +34,57 @@ describe("Artist Top Works Rail Component", () => {
     expect(component.find(Carousel)).toHaveLength(1)
     expect(component.find(FillwidthItem)).toHaveLength(2)
   })
+
+  it("tracks the analytics properties when an artwork is clicked on the Top Works rail", () => {
+    const component = mount(<ArtistTopWorksRail {...props} />)
+    const elem = component.find(FillwidthItem).first()
```

Comment:
i think adding `as FillwidthItem` might kill this type error

Reply:
Good catch! Just updated it.